handler/util.go

```diff
@@ -0,0 +1,75 @@
+package handler
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/vkhichar/asset-management/contract"
+	"github.com/vkhichar/asset-management/service"
+)
+
+var (
+	ErrMissingToken = errors.New("missing token")
+	ErrInvalidToken = errors.New("invalid or expired token")
+	ErrForbidden    = errors.New("forbidden")
+	ErrBadRequest   = errors.New("bad request")
+)
+
+func ReadTokenFromRequest(r *http.Request) (string, error) {
```

Comment:
move out of 'utils.go' file and create a separate file. this doesn't seems an utility

Reply:
Probably write a middleware and add that middleware to router level